src/main/java/com/alseed/todolist/Main.java

```diff
@@ -54,32 +63,27 @@ private static int getCurrentId() {
 
     private static String print(String[] input){
         String output;
+        StringBuilder stringBuilder = new StringBuilder();
         try {
             String command;
             if (input.length==1) {
-                String[] finalOutput = new String[] {""};
                 taskList.
                         stream()
-                        .filter((s) -> !s.isToggled())
-                        .forEach((s) -> finalOutput[0]+=printOutputCreator(s));
-                return finalOutput[0];
+                        .filter((s) -> !s.isCompleted())
+                        .forEach((s) -> stringBuilder.append(printOutputCreator(s)));
```

Comment:
Вообще, если уж очень хочется получить в результате команды строку, чтобы потом её печатать, то я  бы сделал примерно следующее
```suggestion
                        String out = taskList.
                                       stream()
                                       .filter((s) -> !s.isCompleted())
                                       .map(Main::printOutputCreators)
                                       .collect(Collectors.toString("\n"));
                         return out;
```
И убрать из printOutputCreators перевод на новою строку, чтобы не было лишних строк.

Reply:
```
                String out = taskList.
                        stream()
                        .filter((s) -> !s.isCompleted())
                        .map(Main::printOutputCreator)
                        .collect(Collectors.joining("\n"));
```
Получилось сделать таким образом. Понял, как оно работает. 
Но почему нельзя использовать мой вариант? Этот красивее, или более правильный?

Reply:
Более корректное использование StreamAPI